inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/HiveLoadNode.java

```diff
@@ -37,6 +37,9 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * hive load node with FlinkCdc
```

Comment:
Does hive node belong to flink cdc? Is it more appropriate to belong to flink connector?

Reply:
sure it's a typo, thanks